grl/environment/pomdp_files/tiger-alt.POMDP

```diff
@@ -64,7 +64,21 @@ R: open-right : r1 : * : * -100
 R: open-right : r2 : * : * -100
 
 Pi_phi:
-1 1 0
-1 2 0
-2 1 0
-2 2 0
+0 1 0
+0 1 0
+0 0 0
+
+Pi_phi:
+0 0 1
+0 0 1
+0 0 0
+
+Pi_phi:
+0 1 0
+0 0 1
+0 0 0
+
+Pi_phi:
+0 0 1
+0 1 0
+0 0 0
```

Comment:
So now we have a `Pi_phi` listing for every policy we want to check?

Reply:
Yep, it was the easiest way to do it.

Reply:
If there is a different way that seems better, that's cool too, i don't care about this one in particular. Just comes down to parsing and figured it works. Could also imagine like separating by double new line or something